packages-exp/auth-exp/src/core/user/token_manager.ts

```diff
@@ -76,6 +69,17 @@ export class StsTokenManager {
   async getToken(
     auth: AuthInternal,
     forceRefresh = false
+  ): Promise<string | null> {
+    if (!this.isPassthroughMode) {
+      return this.getTokenAndRefreshIfNeeded(auth, forceRefresh);
+    } else {
+      return this.getTokenAndTriggerCallback(/* auth, forceRefresh */);
```

Comment:
why the comment?

Reply:
I believe as a reminder for the next PR which will introduce these parameters. (@lisajian correct me if I'm wrong)

Reply:
Yeah! Sorry that it was unclear - the getTokenAndTriggerCallback will take in these args, but I was getting compiler errors when I left them as part of the method signature since they're currently unused/unimplemented. See future PR for when these are used

Reply:
you can put a `_` before the variable name to signify it is unused https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#flag-unused-declarations-with---nounusedparameters-and---nounusedlocals

Reply:
+1 and the warning should be addressed within `getTokenAndTriggerCallback`, not here

Reply:
Sick, done!